os/manual-rollbacks.md

```diff
@@ -2,6 +2,8 @@
 
 As much as we all love flawless, automatic updates, there may be occasions when an update must be rolled back. This is fairly straightforward on Container Linux, once you know the right commands.
 
+**Note**: Container Linux allows you to rollback to a previously active partition, or change to an older release channel (to downgrade to a stable version). It does not allow you to downgrade to an uninstalled, user defined version number.
```

Comment:
@zbwright Just to recite the cases in case we've caused confusion:

Suppose: A new version is released on the Alpha channel and my Container Linux node updates.

1. If the update fails, automatic rollback to B partition (supported; automatic)
OR
2. Update succeeds and I run on with the new version number (supported)
OR
3. Update succeeds, but I roll back to whatever version is on my B partition as described in this doc (you're on your own)
OR
4. Update succeeds, but I choose a lower channel and "update down" to whatever version number is current on that channel (unsupported)